src/org/labkey/test/pages/wiki/EditPage.java

```diff
@@ -116,6 +116,12 @@ public EditPage addAttachment(File file)
         return this;
     }
 
+    public EditPage clickShowPageTree()
+    {
+        elementCache().showPageTreeButton.click();
+        return this;
+    }
```

Comment:
Wait for tree to be visible.
```suggestion
    public EditPage clickShowPageTree()
    {
        elementCache().showPageTreeButton.click();
        shortWait().until(ExpectedConditions.visibilityOfElementLocated(Locator.id("wiki-toc-tree")));
        return this;
    }
```